mailpoet/assets/js/src/newsletters/send.tsx

```diff
@@ -322,6 +322,13 @@ class NewsletterSendComponent extends Component<
           thumbnailPromise,
           validationError: validateNewsletter(response.data),
         });
+
+        if (response.data?.wp_post_id) {
+          MailPoet.trackEvent(
+            'New Email Editor > Newsletter created by BlockEmailEditor',
+          );
+        }
```

Comment:
I see that this event is tracked every time the send page is loaded. So if user reloads the page or return back to the editor and re-enters the page we track `'New Email Editor > Newsletter created by BlockEmailEditor'`. I'm not sure what to suggest here. Perhaps, could we track on user action when a user clicks "Send"?

Reply:
Hmm, the idea behind this section is to track when a newsletter created by the new editor gets to the Send page. 

I can update the wording, but this entails knowing that users find value in the editor. They might or might not be able to send a newsletter, but the fact that they got to this page is encouraging. 

Does this make sense, @costasovo?

Reply:
> Does this make sense, @costasovo?

Yes, it makes sense! With the current wording, seeing the event without the code context, I would assume it is tracked when a newsletter is created. So perhaps a different wording isn't a bad idea. Would `'New Email Editor > Send page accessed'` work?

Reply:
oppss, I saw your comment late.
Fixed in https://github.com/mailpoet/mailpoet/pull/6003/commits/8f41e1fa7d24b875bef2df32ab3ba1395cc85123